### ISSUE TITLE
Iron+Gold: Raw Ore -> Tiny Dust -> Hard Ore yields too much ore

### ISSUE THREAD

**Issue by cesarmalari** [Beyond-Reality/Beyond-Reality-Divergence#82]
For all the ores, Raw Ore -> Tiny Dust (grinder or windmill) is 1:2, and the Sifter can take that 8:1 to make the full dust, making Raw Ore -> Dust a 4:1.  Using the Pressure Packager (9:1 to make Hard Ore) and the Smelter (1:2 to make an ingot), that becomes 9:2, but faster, but Iron and Gold don't support that.

One of the recent updates added a Tiny Dust -> Hard Ore recipe to the Pressure Packager at 9:1, which means if you use windmill -> Pressure Packager -> Smeltery, you can turn 9 Raw Iron/Gold Ore into 4 Ingots.  I'm guessing that should have been Raw Ore as the input, not Tiny Dust.

**Comment by cesarmalari** [comment]
tl;dr: instead of adding a 9 Tiny Dust -> 1 Hard Ore to the Pressure Packager for Iron/Gold, it should be 9 Raw Ore -> 1 Hard Ore.

**Comment by b0bst3r** [comment]
Yeah the raw iron and gold ores will not go in the pressure packer.

One for @draco18s

**Comment by Draco18s** [comment]
Heh, a few more instances where something got moved and which messed up the variable-reuse and was using the wrong item stack definition.

Fixed for next version.
